docs/source/introduction.rst

```diff
@@ -2,43 +2,162 @@
+Clusters of Sawtooth nodes can be easily deployed with separate permissioning.
+There is no centralized service that could potentially leak transaction
+patterns or other confidential information.
+
+The blockchain stores the settings that specify the permissions, such as roles
+and identities, so that all participants in the network can access this
+information.
+
+Parallel transaction execution
+------------------------------
+
+Most blockchains require serial transaction execution in order to guarantee
+consistent ordering at each node on the network. Sawtooth includes an advanced
+parallel scheduler that splits transactions into parallel flows. Based on the
+locations in state which are accessed by a transaction, Sawtooth isolates the
+execution of transactions from one another while maintaining contextual
+changes.
+
+When possible, transactions are executed in parallel, while preventing
+double-spending even with multiple modifications to the same state. Parallel
+scheduling provides a substantial potential increase in performance over
+serial execution.
+
+Event system
+------------
+
+Hyperledger Sawtooth supports creating and broadcasting events. This allows
+applications to:
+
+    * Subscribe to events that occur related to the blockchain, such as a new
+      block being committed or switching to a new fork.
+
+    * Subscribe to application specific events defined by a transaction family.
+
+    * Relay information about the execution of a transaction back to clients
+      without storing that data in state.
+
+Subscriptions are submitted and serviced over a ZMQ Socket.
+
+Ethereum contract compatibility with Seth
+-----------------------------------------
+
+The Sawtooth-Ethereum integration project, Seth, extends the interoperability
+of the Sawtooth platform to Ethereum. EVM (Ethereum Virtual Machine) smart
+contracts can be deployed to Sawtooth using the Seth transaction family.
+
+Pluggable consensus algorithms
+------------------------------
+
+In a blockchain, consensus is the process of building agreement among a group
+of mutually distrusting participants. Algorithms for achieving consensus with
+arbitrary faults generally require some form of voting among a known set of
+participants. General approaches include Nakamoto-style consensus, which
+elects a leader through some form of lottery, and variants of the traditional
+`Byzantine Fault Tolerance (BFT)
+<https://en.wikipedia.org/wiki/Byzantine_fault_tolerance>`_
+algorithms, which use multiple rounds of explicit votes to achieve consensus.
+
+Sawtooth abstracts the core concepts of consensus and isolates consensus from
+transaction semantics. The interface supports plugging in various consensus
+implementations. More importantly, Sawtooth allows different types of
+consensus on the same blockchain. The application can choose at compile time
+or change on a running blockchain with a transaction.
```

Comment:
[ SHAWN ] Change to:
   "The consensus is selected during the initial network setup and can be changed on a running blockchain with a transaction."